static/js/click_handlers.js

```diff
@@ -308,18 +308,27 @@ export function initialize() {
         popovers.hide_all();
     });
     $("body").on("click", ".always_visible_topic_edit,.on_hover_topic_edit", function (e) {
+        // Hide the pencil icon after clicking on it to
+        // handle the case of the topic which is not set
+        $(".always_visible_topic_edit").hide();
```

Comment:
I agree that if you scope these to the target `recipient_row` (which we already lookup in the next line), that should fix the bugs.

Reply:
Done